ui/components-lib/src/components/HistoryView/_HistoryView.scss

```
@@ -306,34 +307,92 @@
   padding-top: 0;
 }
 
+// Commit column header: primary COMMIT label beside a lowercase accent
+// "promotes" caption whose trailing arrow points into the first env pill.
+.hp-matrix__head-commit {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  align-self: stretch;
+
+  position: sticky;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)